src/osd/PG.h

```diff
@@ -268,6 +268,19 @@ class PG : public DoutPrefixProvider, public PeeringState::PeeringListener {
       });
   }
 
+  static void add_objects_scrubbed_count(
+    int64_t count, pg_stat_t &stats) {
+    stats.objects_scrubbed += count;
+  }
+
+  void add_objects_scrubbed_count(int64_t count) {
+    recovery_state.update_stats(
+      [=](auto &history, auto &stats) {
+	add_objects_scrubbed_count(count, stats);
```

Comment:
Here is the only real issue I find in this PR:
the 'true' here means that we are publishing the stats for every
chunk. Isn't it a waste? could we just collect the numbers,
and publish when all the other stats are published at scrub_finish()?

Reply:
Yeah that makes sense. I'll make the change.

Reply:
@ronen-fr I think publishing stats for every chunk makes sense so that this stat can give us an idea of how much progress the scrub has made, would you agree?